android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1424,7 +1424,12 @@
 	 * @module.api
 	 */
 	public static final String PROPERTY_FONT_SIZE = "font-size";
-
+	
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_MINIMUM_FONT_SIZE = "minimumFontSize";
```

Comment:
keeping in format with TiC:
+ properties are in alphabetical order so place after [PROPERTY_MIN_AGE](https://github.com/hansemannn/titanium_mobile/blob/d054b18df81e65f6b7ca165eb548476fb8c1e42a/android/titanium/src/java/org/appcelerator/titanium/TiC.java#L1890).
+ refactor to `PROPERTY_MIN_FONT_SIZE` like `PROPERTY_MIN_AGE`. not sure why `PROPERTY_MIN_AGE`, `PROPERTY_MIN_UPDATE_DISTANCE`, `PROPERTY_MIN_UPDATE_TIME` do not have `@module.api` tag.